packages/cnb_app_lifecycle/packaging

```diff
@@ -0,0 +1,34 @@
+set -e
+
+# create GOPATH
+mkdir -p ${BOSH_INSTALL_TARGET}/src
+mv * ${BOSH_INSTALL_TARGET}/src
+mv ${BOSH_INSTALL_TARGET}/src .
+
+DEST="$PWD/bin"
+mkdir -p ${DEST}
+
+source /var/vcap/packages/golang-*-linux/bosh/compile.env
+
+pushd src/code.cloudfoundry.org
+
+CGO_ENABLED=0 go build -o ${DEST}/ -ldflags "-s -w" -a -installsuffix static code.cloudfoundry.org/cnbapplifecycle/cmd/builder code.cloudfoundry.org/cnbapplifecycle/cmd/launcher
+
+GOOS=windows CGO_ENABLED=0 go build -o ${DEST}/builder.exe -ldflags "-s -w" -a -installsuffix static code.cloudfoundry.org/buildpackapplifecycle/builder
+GOOS=windows CGO_ENABLED=0 go build -o ${DEST}/launcher.exe -ldflags "-s -w" -a -installsuffix static code.cloudfoundry.org/buildpackapplifecycle/launcher
+popd
+
+ldd $DEST/builder && echo "builder must be statically linked" && false
+ldd $DEST/launcher && echo "launcher must be statically linked" && false
+
+cp /var/vcap/packages/diego-sshd/diego-sshd ${DEST}/diego-sshd
+cp /var/vcap/packages/diego-sshd/*.exe ${DEST}
+cp /var/vcap/packages/diego-sshd/winpty.dll ${DEST}/winpty.dll
+cp /var/vcap/packages/healthcheck/healthcheck ${DEST}/healthcheck
+cp /var/vcap/packages/healthcheck/healthcheck ${DEST}/healthcheck.exe
+
+tar -czf ${BOSH_INSTALL_TARGET}/cnb_app_lifecycle.tgz \
+  -C ${DEST} \
+  builder launcher healthcheck diego-sshd \
+  builder.exe launcher.exe healthcheck.exe diego-sshd.exe \
+  winpty-agent.exe winpty.dll
```

Comment:
```suggestion
cp /var/vcap/packages/diego-sshd/diego-sshd ${DEST}/diego-sshd
cp /var/vcap/packages/healthcheck/healthcheck ${DEST}/healthcheck

tar -czf ${BOSH_INSTALL_TARGET}/cnb_app_lifecycle.tgz \
  -C ${DEST} \
  builder launcher healthcheck diego-sshd
```

Reply:
since Windows is not supported, we dont need to include the packages for Windows platform

Reply:
Sounds reasonable. We just wanted to keep it in synch with `buildpacks_app_lifecycle`.